### DISCUSSION TITLE
IKEA Styrbar (E2001/E2002) not registering hold or release clicks for left/right

### DISCUSSION THREAD

**Discussion by mauritsivs** [Koenkk/zigbee2mqtt#14881] [Devices]
I have succesfully paired an IKEA Styrbar remote. according to Z2M docs it should have mappings for left and right arrow for HOLD and RELEASE just like it has for the up/down buttons.

mine does not register those. it only registers single clicks.
(the up/down button DOES register the hold and release triggers)

anybody that has this same remote working with the correct function for hold & release?

my Z2M versions:
Zigbee2MQTT version [1.27.2]
commit: [7dc48fb]
Coordinator type
: zStack3x0
Coordinator revision: 20210708
Frontend version
: 0.6.110

**Comment by mauritsivs** [comment]
nevermind! it actually works, but you have to press the button for several seconds (3 secs at elast?) before it registers the long-press. which makes the function quite useless ofcourse. oh well....

**Reply by muxator** [reply]
> you have to press the button for several seconds (3 secs at elast?) before it registers the long-press

I wonder if there is a way to programmatically set that delay.
